lib/samples/filter_by_definition_expression_or_display_filter/filter_by_definition_expression_or_display_filter_sample.dart

```diff
@@ -27,48 +27,43 @@ class FilterByDefinitionExpressionOrDisplayFilterSample extends StatefulWidget {
 
 class _FilterByDefinitionExpressionOrDisplayFilterSampleState
     extends State<FilterByDefinitionExpressionOrDisplayFilterSample> {
+  // create a map view controller
   final _mapViewController = ArcGISMapView.createController();
   final _featureLayer = FeatureLayer.withFeatureTable(
       ServiceFeatureTable.withUri(Uri.parse(
           'https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/rest/services/SF_311_Incidents/FeatureServer/0')));
-
-  @override
-  void initState() {
-    super.initState();
-
-    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
-    map.operationalLayers.add(_featureLayer);
-    map.initialViewpoint = Viewpoint.withLatLongScale(
-      latitude: 37.7759,
-      longitude: -122.45044,
-      scale: 100000,
-    );
-
-    _mapViewController.arcGISMap = map;
-  }
+  var displayFilterDefinition;
```

Comment:
```suggestion
  ManualDisplayFilterDefinition? displayFilterDefinition;
```
This is currently a dynamic, needs a type specifying

Reply:
Okay! But shouldn't we be using type inference instead of specifying the type directly?

Reply:
Yes - but it isn't smart enough to do that here, so it infers `dynamic` which we don't want, we want it to be a `ManualDisplayFilterDefinition`. 
If we were instantiating it here like `var displayFilterDefinition = ManualDisplayFilterDefinition()` then it would be ok.